package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
```

Comment:
We need to add types and rename `name` to let it works
```suggestion
  "name": "change-me",
  "types": "types/index.d.ts",
  "scripts": {
```

Reply:
we had it already above. so i patched it

Reply:
I got this:

```
The type definition `index.d.ts` does not exist at `/Users/mspigolon/workspace/_experiments/fastify-plugin-template/index.d.ts`. Is the path correct? Create one and try again.
```

Reply:
fixed